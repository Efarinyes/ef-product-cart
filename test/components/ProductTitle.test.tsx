import React from 'react';
import renderer from 'react-test-renderer';
import { ProductTitle, ProductCard } from '../../src/components';
import { product1 } from '../data/products';


describe('ProductTitle', () => {
    test('Ha de mostrar el component de manera correcta i amn el tÃ­tol personalitzat', () => {
        const wrapper = renderer.create(
            <ProductTitle title="Custom Product" />
        )
        expect(wrapper.toJSON()).toMatchSnapshot();
    });

    test('Ha de mostrar el component amb el nom del producte', () => {
        const wrapper = renderer.create(
            <ProductCard product={product1}>
                {
                    () => (
                        <ProductTitle />
                    )
                }
            </ProductCard>
        )
        expect(wrapper.toJSON()).toMatchSnapshot();
    })

})
