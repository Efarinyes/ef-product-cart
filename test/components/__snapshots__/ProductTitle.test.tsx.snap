// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitle Ha de mostrar el component amb el nom del producte 1`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription undefined"
  >
     
    Coffee Mug Card
     
  </span>
</div>
`;

exports[`ProductTitle Ha de mostrar el component de manera correcta i amn el t√≠tol personalitzat 1`] = `
<span
  className="productDescription undefined"
>
   
  Custom Product
   
</span>
`;
